apps/L1G/tinder/tinder-frontend/src/app/(auth)/signup/page.tsx

```
@@ -1,12 +1,15 @@
-import { TinderLogo } from '@/components/TinderLogo';
+import { GenderSelect } from '@/components/GenderSelect';
+import { MainHeader } from '@/components/MainHeader';
 import YourDetailsPage from '@/components/YourDetailsPage';
```

ESLint

[error] 'YourDetailsPage' is defined but never used.

(no-unused-vars)